backend-api/src/functions/tests/pact/asyncCredential/asyncCredentialConfiguration.ts

```diff
@@ -0,0 +1,68 @@
+import { IAsyncCredentialDependencies } from "../../../asyncCredential/handlerDependencies";
+import { registeredLogs } from "../../../asyncCredential/registeredLogs";
+import { MockEventWriterSuccess } from "../../../services/events/tests/mocks";
+import { Logger } from "../../../services/logging/logger";
+import { MockLoggingAdapter } from "../../../services/logging/tests/mockLogger";
+import {
+  MockClientRegistryServiceGetPartialClientSuccessResultIPV,
+  MockSessionServiceCreateSessionSuccessResult,
+  MockTokenServiceInvalidSignatureIPV,
+  MockTokenServiceSuccessIPV,
+} from "../../../testUtils/asyncCredentialMocks";
+
+const env = {
+  SIGNING_KEY_ID: "mockKid",
+  ISSUER: "mockIssuer",
+  SESSION_TABLE_NAME: "mockTableName",
+  SESSION_TABLE_SUBJECT_IDENTIFIER_INDEX_NAME: "mockIndexName",
+  SESSION_TTL_IN_MILLISECONDS: "12345",
+  SQS_QUEUE: "mockSqsQueue",
+  CLIENT_REGISTRY_PARAMETER_NAME: "mockParmaterName",
+};
+
+export class AsyncCredentialConfiguration {
```

Comment:
Hello - I'm not tied to a particular pattern but I still think this is overly complicated and hard to work out what's going on